lib/wisper/activerecord/publisher.rb

```diff
@@ -16,11 +16,8 @@ module Publisher
       included do
         include Wisper::Publisher
 
-        # NOTE: do not need to silence deprecations on Rails 5+
```

Comment:
I'm +1 for removing this deprecation silence.  Could you update the gemspec to now enforce Rails 5+ going forward?

Reply:
Good call, done!